media-sound/bitwig-studio/bitwig-studio-4.3.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop unpacker xdg
+
+DESCRIPTION="Multi-platform music-creation system for production, performance and DJing"
+HOMEPAGE="http://bitwig.com"
+SRC_URI="https://downloads.bitwig.com/stable/${PV}/${P}.deb"
+LICENSE="Bitwig"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+IUSE="abi_x86_32 +jack cpu_flags_x86_sse4_1"
+REQUIRED_USE="cpu_flags_x86_sse4_1"
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	app-arch/bzip2
+	dev-libs/expat
+	gnome-extra/zenity
+	jack? ( media-sound/jack2 )
```

Comment:
Is `media-sound/jack2` really required or would `virtual/jack` do as well? Most applications are compatible with pipewire, so you'd usually want to depend on `virtual/jack` which can be provided by `media-sound/pipewire[jack-sdk]`.

Reply:
FYI https://github.com/gentoo-audio/audio-overlay/pull/76#issuecomment-335279957

Could be this is fixed in Bitwig now (or that symbol is now also included/exposed in JACK), haven't checked

Reply:
Hmm okay. That still doesn't rule out media-sound/pipewire[jack-sdk] as a potential candidate. Afaik its libjack.so.0 is fully ABI compatible with JACK 2's libjack.so.0. So maybe `|| ( media-sound/jack2 media-video/pipewire[jack-sdk] )` is the correct dependency here.

Reply:
In the main tree, only jack-example-tools/jack-example-tools-4.ebuild depend on media-sound/jack2[libsamplerate], no package depend on media-sound/pipewire and around 70 or 80 software depend on virtual/jack. I would be very surprised is the correct depend for bitwig-studio is not virtual/jack. Also, to put a hard dependency on jack2 would break the virtual, which imply than if it is no imperative reason to break it, the ebuild should use virtual/jack.